examples/basic_use.py

```diff
@@ -51,4 +50,4 @@
 # Linking a remote jira issue (needs applinks to be configured to work)
 issue = jira.issue("JRA-1330")
 issue2 = jira.issue("XX-23")  # could also be another instance
-jira.add_remote_link(issue, issue2)
+jira.add_remote_link(str(issue.id), issue2)
```

Comment:
why cast to str? issue.id is a string already..

Reply:
Yep you are completely right. The original typehint was an `Optional[int]` which was incorrect, it is always present, and is a `str`. I verified this both from the API documentation and by checking examples in jira.atlassian.com 

It also helped me remove the assertion in this file